docs/fundamentals/code-analysis/style-rules/ide0280.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "IDE0280: Use 'nameof'"
+description: "Learn about code analysis rule IDE0280: Use 'nameof'"
+ms.date: 07/26/2023
+ms.topic: reference
+f1_keywords:
+- IDE0280
+helpviewer_keywords:
+- IDE0280
+dev_langs:
+- CSharp
+---
+# Use 'nameof' (IDE0280)
+
+| Property                 | Value                                         |
+|--------------------------|-----------------------------------------------|
+| **Rule ID**              | IDE0280                                       |
+| **Title**                | Use 'nameof'                                  |
+| **Category**             | Style                                         |
+| **Subcategory**          | Language rules (parameter preferences)        |
```

Comment:
I classified it under "parameter preferences" but also considered "expression-level preferences".